setup.py

```diff
@@ -121,7 +121,7 @@
             'ssdutil = ssdutil.main:ssdutil',
             'pfc = pfc.main:cli',
             'psuutil = psuutil.main:cli',
-            'fwutil = fwutil.main:cli',
```

Comment:
don't remove this line. this is a change that went in a few days ago. please resolve the conflicts properly

Reply:
i sorry for that mistake , i will update it right away

Reply:
i am sorry for that mistake ,i will update it right away